code/Controllers/CMSMain.php

```diff
@@ -2114,23 +2114,23 @@ public function batchactions()
 
     public function BatchActionParameters()
     {
-        $batchActions = CMSBatchActionHandler::config()->batch_actions;
+        $batchActions = $this->batchactions()->registeredActions();
 
         $forms = [];
         foreach ($batchActions as $urlSegment => $batchAction) {
-            $SNG_action = singleton($batchAction);
-            if ($SNG_action->canView() && $fieldset = $SNG_action->getParameterFields()) {
+            $SNG_action = singleton($batchAction["class"]);
+            if ($SNG_action->canView() && $fieldList = $SNG_action->getParameterFields()) {
                 $formHtml = '';
                 /** @var FormField $field */
-                foreach ($fieldset as $field) {
-                    $formHtml .= $field->Field();
+                foreach ($fieldList as $field) {
+                    $formHtml .= $field->FieldHolder();
                 }
                 $forms[$urlSegment] = $formHtml;
             }
         }
         $pageHtml = '';
         foreach ($forms as $urlSegment => $html) {
-            $pageHtml .= "<div class=\"params\" id=\"BatchActionParameters_$urlSegment\">$html</div>\n\n";
+            $pageHtml .= "<div class=\"params\" id=\"BatchActionParameters_$urlSegment\" style=\"display:none\">$html</div>\n\n";
```

Comment:
```suggestion
            $pageHtml .= '<div class="params" id="BatchActionParameters_' . $urlSegment . '" style="display:none">' . $html . '</div>';
```
I think this is a little easier to read, and as a bonus it matches the style of the literalfield below.